openshift/fluent-bit.cm.yaml

```diff
@@ -102,7 +194,17 @@ parameters:
     description: Target namespace reference (i.e. 'b160aa-dev')
     displayName: Target Namespace
     required: true
-  - name: LOGGING_HOST_NAME
-    description: The hostname of our FLuentd service running further down the monitoring pipeline
-    displayName: Logging host name
+  - name: FLUENTD
+    description: The hostname of our Fluentd service running further down the monitoring pipeline
+    displayName: Fluentd host name
     required: true
+  - name: AWS_DEFAULT_REGION
+    description: Default AWS region for Opensearch logging stack
+    displayName: AWS Default Region
+  - name: AWS_KINESIS_STREAM
+    description: AWS Kinesis Stream identifier
+    displayName: AWS Kinesis Stream
+  - name: AWS_ROLE_ARN
+    description: AWS OpenSearch/Kinesis Resource Name
+    displayName: AWS Role ARN
```

Comment:
Required attributes should be added to these, and then the example command in the readme should have these in there too.